code/code/disc/disc_adv_adventuring.cc

```diff
@@ -834,8 +834,6 @@ void TBeing::doSeekwater()
   }
 
   // Let's determine the distance:
-  // Ranges: 21, ..., 120
-  dist = (max(5, lev)/5) * (max(10, skill)/10) + 20;
   dist = lev * max(10, skill);
```

Comment:
I'm just removing dead code. If you want to review whether dist is actually getting a good number, we can open a separate ticket.